### PR TITLE
Customize TinyMCE "iframe sandboxing" behavior

### LINKED ISSUE #1414: Documentation: TinyMCE 7.0 defaults `sandbox_iframes` to `true`
If you put an `<iframe>` into TinyMCE it now is sandboxed per default as of version 7. Additionally there is the `sandbox_iframes_exclusions` (https://www.tiny.cloud/docs/tinymce/latest/content-filtering/#sandbox-iframes-exclusions) option which defines URLs not to be sandboxed and defaults to:

```
[
  'youtube.com',
  'youtu.be',
  'vimeo.com',
  'player.vimeo.com',
  'dailymotion.com',
  'embed.music.apple.com',
  'open.spotify.com',
  'giphy.com',
  'dai.ly',
  'codepen.io'
]
```

Now if you want to have other URLs than these, you could use `Other settings` in the `Advanced` tab of the TinyMCE controlpanel and add either

```
{
    "sandbox_iframes": false
}
```

to deactivate sandboxing or extend the exclusion list with
```
{
    "sandbox_iframes_exclusions": [
        // default urls ...
        "my.url.com",
    ]
}
```

I propose to add this to the documentation here https://6.docs.plone.org/classic-ui/tinymce-customization.html

/cc @plone/classicui-team

### PR DESCRIPTION
**Pull request by petschki**
Fixes https://github.com/plone/mockup/issues/1414

----
📚 Documentation preview 📚: https://plone6--1814.org.readthedocs.build/

### CODE REVIEW

File: docs/classic-ui/tinymce-customization.md [hunk 1]
```diff
@@ -138,3 +138,31 @@ You can remove these formats through the TinyMCE control panel.
...
 
 Once removed, the custom formats will no longer appear in the menu.
+
+
+
+## Configure `<iframe>` sandboxing
+
+Since version 7.0, TinyMCE adds the attribute `sandbox=""` to make the `<iframe>` elements [sandboxed](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox) with all restrictions.
+
+To customize this behavior there are two options which can be added to the `Other settings` JSON configuration in the `Advanced` tab of the TinyMCE controlpanel.
```
Comment (stevepiercy) [inline docs/classic-ui/tinymce-customization.md]:
```suggestion
To customize this behavior, there are two options which you can add to the {guilabel}`Other settings` JSON configuration in the {guilabel}`Advanced` tab of the {guilabel}`TinyMCE` control panel.
```

File: docs/classic-ui/tinymce-customization.md [hunk 2]
```diff
@@ -138,3 +138,31 @@ You can remove these formats through the TinyMCE control panel.
...
+
+If you want to deactivate sandboxing in general this is the JSON configuration:
```
Comment (stevepiercy) [inline docs/classic-ui/tinymce-customization.md]:
```suggestion
If you want to deactivate sandboxing in general, use the following JSON configuration.
```

File: docs/classic-ui/tinymce-customization.md [hunk 3]
```diff
@@ -138,3 +138,31 @@ You can remove these formats through the TinyMCE control panel.
...
+
+```JSON
+{
+  "sandbox_iframes": false
+}
+```
+
+You can also exclude certain URLs from beeing sandboxed. The option is:
```
Comment (stevepiercy) [inline docs/classic-ui/tinymce-customization.md]:
```suggestion
You can also exclude certain URLs from being sandboxed as follows.
```

File: docs/classic-ui/tinymce-customization.md [hunk 4]
```diff
@@ -138,3 +138,31 @@ You can remove these formats through the TinyMCE control panel.
...
+
+```JSON
+{
+  "sandbox_iframes_exclusions": [
+    "my.url.com"
+  ]
+}
+```
+
+See [this article](https://www.tiny.cloud/docs/tinymce/latest/content-filtering/#sandbox-iframes-exclusions) for TinyMCEs default settings.
```
Comment (stevepiercy) [inline docs/classic-ui/tinymce-customization.md]:
````suggestion
```{seealso}
See [`sandbox_iframes_exclusions`](https://www.tiny.cloud/docs/tinymce/latest/content-filtering/#sandbox-iframes-exclusions) for TinyMCE's default settings.
```
````

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Sprinkle some MyST seasoning to taste, stir in some grammar and spelling, and it's good to go. Thank you!

**Review by stevepiercy — APPROVED** [review]

**Comment by stevepiercy** [comment]
Direct link to changes, but wait until it finishes its build.
https://plone6--1814.org.readthedocs.build/classic-ui/tinymce-customization.html#configure-iframe-sandboxing

**Comment by petschki** [comment]
Thanks ! Looking good!

**Comment by stevepiercy** [comment]
We can ignore the read timeouts. See #1815 and #1816.

**Pull request merged**